### PR TITLE
Remove select2-rails

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.12
* [ ] 1.11
* [ ] 1.10

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---
Depends on theforeman/foreman#3736/

### CODE REVIEW

**Comment by domcleal** [comment]
Also add select2-rails to the Obsoletes list in tfm/tfm.spec please.

**Comment by dLobatog** [comment]
[test] - it's on the tfm spec obsoletes now

File: tfm/tfm.spec [hunk 1]
```diff
@@ -73,6 +73,7 @@ Obsoletes: %{scl_prefix}rubygem-multi_json < 1.10.1-4
 Obsoletes: %{scl_prefix}rubygem-nokogiri < 1.6.6.2-3
 Obsoletes: %{scl_prefix}rubygem-sass < 3.4.19-4
 Obsoletes: %{scl_prefix}rubygem-sass-rails < 5.0.4-3
+Obsoletes: %{scl_prefix}rubygem-select2-rails = 3.5.10-2
```
Comment (domcleal) [inline tfm/tfm.spec]: This should be `< 3.5.10-3` to match the current and all previous versions.

**Comment by dLobatog** [comment]
Updated

**Comment by domcleal** [comment]
Thanks @dLobatog.

**Pull request merged**